spec

```diff
@@ -2785,7 +2785,6 @@ endseg
 beginseg
     name "ovl_En_Owl"
     include "build/src/overlays/actors/ovl_En_Owl/z_en_owl.o"
-    include "build/data/overlays/actors/z_en_owl.data.o"
     include "build/data/overlays/actors/z_en_owl.reloc.o"
```

Comment:
Auto generated reloc should be used in `NON_MATCHING` builds to avoid reloc issues
```suggestion
#ifdef NON_MATCHING
    include "build/src/overlays/actors/ovl_En_Owl/ovl_En_Owl_reloc.o"
#else
    include "build/data/overlays/actors/z_en_owl.reloc.o"
#endif
```